src/stories/dropdown.js

```diff
@@ -27,20 +27,40 @@ storiesOf('Dropdown', module)
     `,
     () => (
       <div className="uk-margin-bottom">
-        <Dropdown>
+        <Dropdown component="div">
           <button className="uk-button">
             Hover <i className="uk-icon-caret-down" />
           </button>
           <Menu />
         </Dropdown>
         <br /> <br />
-        <Dropdown mode="click">
+        <Dropdown
+          component="div"
```

Comment:
since the default value is 'div' the prop `component` can be removed from this example